x-pack/plugins/security_solution/common/api/timeline/resolve_timeline/resolve_timeline_route.schema.yaml

```diff
@@ -15,7 +15,8 @@ paths:
       x-labels: [serverless, ess]
       x-codegen-enabled: true
       operationId: ResolveTimeline
-      summary: Get an existing saved timeline or timeline template.
+      summary: Resolve a Timeline or Timeline template
+      description: 
```

Comment:
There’s already a 'Get an existing saved timeline or timeline template' operation elsewhere (GET /api/timeline), so the existing summary doesn't seem right. 🤔 Could someone confirm if this suggestion (Resolve a Timeline or Timeline template) would be correct here and help provide an operation description (what does 'resolving' a Timeline mean)?